lib/main.py

```diff
@@ -22,8 +22,33 @@ def send_metrics_to_datadog(metrics, tags={}):
     for metric_name, value in metrics.items():
         dd_client.submit_metric(f'lighthouse.{metric_name}', value, tags)
 
-def main():
+def capture_lighthouse_metrics(page_type, url, audits, lighthouse_options=[]):
     lighthouse = Lighthouse()
+    form_factor = 'mobile'
+
+    if '--preset=desktop' in lighthouse_options:
+        form_factor = 'desktop'
+
+    print(f'Running lighthouse for {url} with {form_factor} options\n')
+    json_results = lighthouse.run(url, lighthouse_options)
+    print(f'Finished running lighthouse for {url}\n')
```

Comment:
The output might look nicer if we tab over this and the following `print` statements.

```
Running lighthouse for {url} with {form_factor} options
   Finished running lighthouse for {url}
   Parsing lighthouse results for {url}
   Sending metrics to datadog for {url}
   Finished sending metrics to datadog for {url}
Running lighthouse for {url} with {form_factor} options
   Finished running lighthouse for {url}
   Parsing lighthouse results for {url}
   Sending metrics to datadog for {url}
   Finished sending metrics to datadog for {url}
Running lighthouse for {url} with {form_factor} options
   Finished running lighthouse for {url}
   Parsing lighthouse results for {url}
   Sending metrics to datadog for {url}
   Finished sending metrics to datadog for {url}

```

Although now I see there is a `print('=' * 80)` so maybe this isn't as important.

Reply:
These print statements are also only seen when running `lighthouse-docker` manually. Otherwise, they are hidden when running it on `ubreakit` (i.e. the server-templates role).